libs/blocks/merch-card-collection/merch-card-collection.js

```diff
@@ -162,6 +162,7 @@ async function initMerchCards(filtered, preferences, cardsRoot) {
   // re-order cards, update card filters
   [...cardsRoot.children].filter((card) => card.tagName === 'MERCH-CARD').forEach((merchCard) => {
     merchCard.style.display = 'none';
+    merchCard.keyHandling = false;
```

Comment:
this seems to be breaking navigation by arrow keys in the catalog page.

Reply:
yes, it just brings navigation back to standard TAB browsing, no custom code at all, (checked with accessibility expert on ticket that seemed ok with it), plus former catalog page didn't have it. Is arrow key browsing for catalog pages (and collections) a requirement from someone else?